sequence_processing_pipeline/NuQCJob.py

```diff
@@ -404,23 +442,7 @@ def _generate_job_script(self, max_bucket_size):
 
         gigabyte = 1073741824
 
-        if max_bucket_size > (3 * gigabyte):
-            gres_value = 4
-        else:
-            gres_value = 2
-
-        # As slurm expects memory sizes to be integer values, convert results
-        # to int to drop floating point component before passing to jinja2 as
-        # a string.
-        if max_bucket_size < gigabyte:
-            mod_wall_time_limit = self.wall_time_limit
-            mod_jmem = str(int(self.jmem))
-        elif max_bucket_size < (2 * gigabyte):
-            mod_wall_time_limit = self.wall_time_limit * 1.5
-            mod_jmem = str(int(self.jmem * 4.5))
-        else:
-            mod_wall_time_limit = self.wall_time_limit * 2
-            mod_jmem = str(int(self.jmem * 7.5))
+        gres_value = 4 if max_bucket_size > (3 * gigabyte) else 2
```

Comment:
Missed this one from the previous review; can you add it to the config file? Thanks